script.js

```diff
@@ -44,11 +44,13 @@ let locations = []
 // See also: https://docs.mapbox.com/mapbox-gl-js/example/setstyle/
 const map = new mapboxgl.Map({
   container: 'map',
-  style: 'mapbox://styles/saman/ckawvg6bk011x1ipepu7nqlbh',
+  style: 'mapbox://styles/saman/ckawvg6bk011x1ipepu7nqlbh?fresh=true',
```

Comment:
Knocks out some caching?

Reply:
Oh I need to remove this param - was getting intermittent 404s due to caching but this param will rate limit is